### PR TITLE
fix(NODE-3272)!: emit correct event type when SRV Polling

### PR DESCRIPTION
**Pull request by nbbeeken**
Corrects the event type emitted from the SRV polling listener on the Topology.

Also:
Skipped the cursor tests and marked them with the unskip ticket, and corrected the visibility on session properties.

### CODE REVIEW

File: test/tools/utils.js [hunk 1]
```diff
@@ -246,12 +246,25 @@ class EventCollector {
   }
 }
 
+function getSymbolFrom(target, symbolName, assertExists = true) {
```
Comment (emadum) [inline test/tools/utils.js:249]: Cool util! 🎉

**Review by emadum — APPROVED** [review]
LGTM! 👍  Looks like this also fixes NODE-3269?

File: test/tools/utils.js (hunk 1 above)
Comment (nbbeeken) [reply test/tools/utils.js:249]: Thanks, and good catch on the other ticket I'll be sure to note it in the merge

File: test/unit/sdam/topology.test.js [hunk 2]
```diff
@@ -314,5 +319,108 @@ describe('Topology (unit)', function () {
...
+              .to.be.a('object')
+              .with.property('address', 'fake:2');
+          });
+
+          topology.s.srvPoller.emit(
+            SrvPoller.SRV_RECORD_DISCOVERY,
+            new SrvPollingEvent([{ priority: 1, weight: 1, port: 2, name: 'fake' }])
+          );
+
+          // The srv event starts a monitor that we don't actually want running
```
Comment (dariakp) [inline test/unit/sdam/topology.test.js:375]: let's move the cleanup into an afterEach and iterate over all servers

**Review by dariakp — CHANGES_REQUESTED** [review]

File: test/unit/sdam/topology.test.js (hunk 2 above)
Comment (nbbeeken) [reply test/unit/sdam/topology.test.js]: Yes, good idea 🚀

**Review by dariakp — APPROVED** [review]
lgtm

**Pull request merged**